projects/fsastorefrontlib/src/cms-components/product/product-list/product-list.component.html

```diff
@@ -90,7 +90,7 @@ <h6 class="title background-color-primary text-white">
           ></cx-pagination>
         </div>
       </div>
-      <div class="col-12 col-md-4 offset-md-2">
+      <div class="col-12 col-md-4 offset-md-2 mt-4 mt-lg-0 mt-md-0">
```

Comment:
Try adding a class `mb-4` to the div above which holds pagination instead of adding this many classes to this div. Also this `offset-md-2` was maybe wrongly added before. It was better to use `ml-auto`